docs/source/posts/2023/2023-01-29-final-report-shivam.rst

```diff
@@ -270,15 +270,15 @@ Timeline
 +----------------------+-----------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | Week 3 (04-07-2022)  | Fixing fetcher adding tests and docs                      | `FURY <https://fury.gl/latest/posts/2022/2022-07-04-week3-shivam.html>`__ - `Python <https://blogs.python-gsoc.org/en/xtanions-blog/week-3-fixing-fetcher-adding-tests-and-docs/>`__                      |
 +----------------------+-----------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| Week 4 (12-07-2022)  | Finalizing glTF loader                                    | `FURY <https://fury.gl/latest/posts/2022/2022-07-12-week4-shivam.html>`__ -`Python<https://blogs.python-gsoc.org/en/xtanions-blog/week-4-finalizing-gltf-loader/>`__                                      |
+| Week 4 (12-07-2022)  | Finalizing glTF loader                                    | `FURY <https://fury.gl/latest/posts/2022/2022-07-12-week4-shivam.html>`__ -`Python <https://blogs.python-gsoc.org/en/xtanions-blog/week-4-finalizing-gltf-loader/>`__                                      |
```

Comment:
Hi @jhlegarreta! 

Good Great.

the two warnings are fixed but there is another one created because of the malformation of the board.

Reply:
![Screenshot from 2024-08-27 22-45-04](https://github.com/user-attachments/assets/56c9f8a7-d09b-44fd-944b-65ca5f432c98)

Reply:
![image](https://github.com/user-attachments/assets/727e8052-75eb-4544-ab29-81da78922e61)

Reply:
:100: Fixed with the last force push. Thanks.